src/components/DirectPaymentCreateModal/DirectPaymentCreateModal.tsx

```diff
@@ -286,142 +309,265 @@ export const DirectPaymentCreateModal: React.FC<DirectPaymentCreateModalProps> =
+          </div>
+
+          <div className="DirectPaymentCreateModal__confirmationDetails">
+            <div className="DirectPaymentCreateModal__confirmationRow">
+              <span className="DirectPaymentCreateModal__confirmationLabel">Amount:</span>
+              <span className="DirectPaymentCreateModal__confirmationValue">
+                {paymentDataToConfirm.amount} {paymentDataToConfirm.asset.code}
+              </span>
+            </div>
+
+            <div className="DirectPaymentCreateModal__confirmationRow">
+              <span className="DirectPaymentCreateModal__confirmationLabel">Receiver:</span>
+              <span className="DirectPaymentCreateModal__confirmationValue">{receiverInfo}</span>
+            </div>
+
+            {walletAddress && (
+              <div className="DirectPaymentCreateModal__confirmationRow">
+                <span className="DirectPaymentCreateModal__confirmationLabel">Wallet:</span>
+                <span className="DirectPaymentCreateModal__confirmationValue">
+                  {selectedWallet?.wallet?.name && (
+                    <span className="DirectPaymentCreateModal__confirmationValueProvider">
+                      {selectedWallet.wallet.name}
+                    </span>
+                  )}
+                  <span className="DirectPaymentCreateModal__confirmationValueAddress">
+                    {shortenAddress(walletAddress)}
+                  </span>
+                </span>
+              </div>
+            )}
+
+            {paymentDataToConfirm.external_payment_id && (
+              <div className="DirectPaymentCreateModal__confirmationRow">
+                <span className="DirectPaymentCreateModal__confirmationLabel">
+                  External Payment ID:
+                </span>
+                <span className="DirectPaymentCreateModal__confirmationValue">
+                  {paymentDataToConfirm.external_payment_id}
+                </span>
+              </div>
+            )}
+          </div>
+        </div>
+      </div>
+    );
+  };
+
   return (
     <Modal visible={visible} onClose={handleClose}>
-      <Modal.Heading>Create a direct payment</Modal.Heading>
-      <form onSubmit={handleSubmit} onReset={handleClose}>
-        <Modal.Body>
-          {errorMessage && (
-            <Notification variant="error" title="Error">
-              <ErrorWithExtras appError={{ message: errorMessage }} />
-            </Notification>
-          )}
-
-          <div className="DirectPaymentCreateModal__form">
-            {/* Asset Selection */}
-            <Select
-              fieldSize="sm"
-              id="assetId"
-              label="Asset"
-              value={formData.assetId}
-              onChange={handleInputChange}
-              error={formErrors.assetId}
-              required
+      <Modal.Heading>
+        {showConfirmation ? "Confirm Direct Payment" : "Create a direct payment"}
```

Comment:
Should we match the title case in both cases to make them consistent?

```suggestion
        {showConfirmation ? "Confirm Direct Payment" : "Create A Direct Payment"}
```